wgpu-hal/src/vulkan/mod.rs

```diff
@@ -455,12 +455,14 @@ pub struct Adapter {
     //queue_families: Vec<vk::QueueFamilyProperties>,
     known_memory_flags: vk::MemoryPropertyFlags,
     phd_capabilities: adapter::PhysicalDeviceProperties,
-    phd_features: adapter::PhysicalDeviceFeatures,
+    phd_features: PhysicalDeviceFeatures,
     downlevel_flags: wgt::DownlevelFlags,
     private_caps: PrivateCapabilities,
     workarounds: Workarounds,
 }
 
+pub use adapter::PhysicalDeviceFeatures;
```

Comment:
Definitely shouldn't put the use here - if anything it should be re-exported at the top of the file, near any other re-exports.